src/components/buttons.js

```diff
@@ -36,34 +37,18 @@ import { SvgIcon } from './SvgIcon';
  * @param {ButtonProps} props
  */
 function ButtonBase({
-  children,
   className,
   icon,
   iconPosition = 'left',
-  disabled,
-  expanded,
-  pressed,
-  onClick = () => {},
   size = 'medium',
-  style = {},
-  title,
   variant = 'normal',
+  expanded,
+  pressed,
+  ...restProps
```

Comment:
I wonder if hiding, e.g., `onClick` in these `restProps` makes it a little bit opaque...

Reply:
Personally, I think it makes a better distinction between the props that are allowed but not directly used or manipulated by `ButtonBase` (`children`, `disabled`, `onClick`, `style`, `title`), which go to `restProps`, from the props that are exclusive of `ButtonBase` (`className`, `icon`, `iconPosition`, `expanded`, `pressed`, `size`)